src/components/10-atoms/input-text/index.js

```diff
@@ -0,0 +1,235 @@
+import { html, svg } from 'lit-element';
+/* eslint-disable import/no-extraneous-dependencies */
+import { FilledTickAnimatedSvg } from '@axa-ch/materials/icons';
+import NoShadowDOM from '../../../utils/no-shadow';
+/* eslint-disable import/no-extraneous-dependencies */
+import defineOnce from '../../../utils/define-once';
+import createRefId from '../../../utils/create-ref-id';
+import styles from './index.scss';
+
+class AXAInputText extends NoShadowDOM {
+  static get tagName() {
+    return 'axa-input-text';
+  }
+
+  static get properties() {
+    return {
+      // Could be Field class
+      id: { type: String },
+      name: { type: String },
+      label: { type: String },
+      placeholder: { type: String },
+      value: { type: String },
+      valid: { type: Boolean },
+      validation: { type: Boolean },
+      // Messages
```

Comment:
if comment is about prop below, move to same line end

Reply:
fix in #27c93be